src/utils/vote-presets.js

```diff
@@ -3,8 +3,40 @@ import { erc20ABI } from 'wagmi'
 import BigNumber from 'bignumber.js'
 
 import VaultABI from './abis/Vault.json'
+import EthPNTABI from './abis/EthPNT.json'
 import settings from '../settings'
 
+const prepareInflationProposal = async (receiverAddress, rawAmount) => {
+  const Vault = new ethers.utils.Interface(VaultABI)
+  const EthPNT = new ethers.utils.Interface(EthPNTABI)
```

Comment:
I think the problem is here too, I'm using these in my code
```js
Vault = await hre.ethers.getContractFactory(VaultAbi, VaultBytecode)
EthPNT = await hre.ethers.getContractFactory(EthPNTAbi, EthPNTBytecode)
```

Reply:
You'll find the bytecode on etherscan

Reply:
[VaultBytecode.txt](https://github.com/pnetwork-association/dao-v2-ui/files/13917340/VaultBytecode.txt)

Reply:
[EthPntBytecode.txt](https://github.com/pnetwork-association/dao-v2-ui/files/13917360/EthPntBytecode.txt)

Reply:
As reviewed in call the problem was relative to the amount used in testing that happend to be in eth and not in wei.
I would close this as not relevant.